docs/BasicCommunication/SetAppProperties/assets/SetAppProperties.txt

```diff
@@ -0,0 +1,47 @@
+title SetAppProperties 
+
+participant Local Web App
+participant SDL
+participant HMI
+
+==<align:center>Pre-conditions: 1. SDL and HMI are started==
```

Comment:
i think these preconditions can be assumed :)

Reply:
@iCollin, do you want me to remove them? or what action is expected?

Reply:
Yes I would remove this precondition but I would keep the other:  `application's properties were set via BC.SetAppProperties from HMI`

Reply:
@iCollin sorry, I don't quite get you. 
this sequence describes how _the properties **are being set**_. 
so why should we have the precondition `application's properties were set via BC.SetAppProperties from HMI`?

so I just remove the existing precondition for this diagram, correct?